kratos/tests/test_model_part_io.py

```diff
@@ -45,6 +45,13 @@ def test_model_part_io_read_model_part(self):
         self.assertEqual(model_part.NumberOfElements(), 4)
         self.assertEqual(model_part.NumberOfConditions(), 5)
 
+        self.assertEqual(model_part[KratosMultiphysics.AMBIENT_TEMPERATURE], 250.0)
+        self.assertEqual(model_part[KratosMultiphysics.DISPLACEMENT_X], 2.1)
+        self.assertEqual(model_part[KratosMultiphysics.DISPLACEMENT_Y], 3.2)
+        self.assertEqual(model_part[KratosMultiphysics.DISPLACEMENT_Z], 4.3)
+        self.assertEqual(model_part[KratosMultiphysics.VELOCITY_X], 3.8)
+        self.assertEqual(model_part[KratosMultiphysics.VELOCITY_Y], 4.9)

```

Comment:
Can you please add a check for the Z component to be 0?

Reply:
I did not know if it would be initialized or uninitialized. If you ask this I assume you know it's initialized to 0.0 by default. I just pushed this change.